backend/open_webui/utils/chat_encryption_proxy.py

```diff
@@ -339,19 +339,40 @@ def get_chat_list_by_user_id(
     def get_archived_chat_list_by_user_id(
         self,
         user_id: str,
-        filter: Optional[dict] = None,
         skip: int = 0,
         limit: int = 50,
     ) -> list[ChatModel]:
         """Override to decrypt chat data after retrieving."""
-        results = super().get_archived_chat_list_by_user_id(
```

Comment:
Why does the previous implementation fail? It looks correct.

The fallback code also looks the same as the previous implementation.

Reply:
the old call incorrectly passed 4 arguments (user_id, filter, skip, limit), the updated one passes only 3 (user_id, skip, limit)